cmd/rig/cmd/capsule/deploy/deploy.go

```diff
@@ -53,6 +56,144 @@ func parseEnvironmentSource(value string) (capsule.EnvironmentSource_Kind, strin
 }
 
 func (c *Cmd) deploy(ctx context.Context, cmd *cobra.Command, args []string) error {
+	if currentFingerprint != "" && currentRolloutID != 0 {
+		return errors.InvalidArgumentErrorf("cannot give both --fingerprint and --current-rollout")
```

Comment:
I believe we can control this in the flags thingy, marking two flags an mutualyl exclusive